tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -384,14 +384,15 @@ public function cleanDataAfterTests(): void {
 	 *
 	 * @throws Exception|GuzzleException
 	 */
-	public function deleteAllSpacesOfTheType(string $driveType): void {
+	public function deleteAllSpacesOfTheType(string $driveType, $remember = true): void {
 		$query = "\$filter=driveType eq $driveType";
 		$userAdmin = $this->featureContext->getAdminUsername();
 
 		for ($i = 0; $i < 2; ++$i) {
 			$this->theUserListsAllHisAvailableSpacesUsingTheGraphApi(
```

Comment:
We should use `theUserListsAllAvailableSpacesUsingTheGraphApi` because we now differentiate between `me/drives` (only the drives where the user has access to) and `drives/`

Reply:
i've pushed the suggested changes, please review again.